src/app/test_executive/test_executive.ml

```diff
@@ -303,6 +303,7 @@ let main inputs =
         let%bind network, dsl =
           Deferred.bind init_result ~f:Malleable_error.of_or_error_hard
         in
+        let%bind () = Engine.Network.initialize ~logger network in
```

Comment:
now that we do this, is it still necessary to wait_for_init at the beginning of every integration test?

Reply:
Yes, this does not wait for init, it just starts the nodes.